backend/Tests/Aggregates/Contract/Types/ModuleSchemaVersionTests.cs

```diff
@@ -0,0 +1,30 @@
+using Application.Aggregates.Contract.Types;
+using FluentAssertions;
+
+namespace Tests.Aggregates.Contract.Types;
+
+public sealed class ModuleSchemaVersionTests
+{
+    [Theory]
+    [InlineData(ModuleSchemaVersion.Undefined, (byte)0)]
+    [InlineData(ModuleSchemaVersion.V0, (byte)0)]
+    [InlineData(ModuleSchemaVersion.V2, (byte)2)]
```

Comment:
Does this turn the test into multiple cases?

Reply:
Yes